lib/github_changelog_generator/project_name_finder.rb

```diff
@@ -0,0 +1,120 @@
+# frozen_string_literal: true
+module GitHubChangelogGenerator
+  class ProjectNameFinder
```

Comment:
I see, makes sense. Sounds a bit like it does two different things that are not even so related: interrogating options on the one hand, parsing a remote on the other hand. Only the thing that comes out of it is the same.